appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.61.5",
+    "version": "0.61.6",
```

Comment:
This is technically a breaking change since you're removing interfaces other extensions were relying on, so let's just bump it up to 0.62.0.